modules/update-upgrading-web.adoc

```diff
@@ -22,18 +27,39 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to the correct
-`stable-<major_version>` channel for the version that you want to update to.
+. For production clusters, ensure that the *CHANNEL* is set to the correct
+`stable-*` channel for
+ifdef::within[]
+the version that you want to update to.
+endif::within[]
+ifdef::between[]
+your current minor version.
+endif::between[]
 +
 [IMPORTANT]
 ====
 For production clusters, you must subscribe to a `stable-*` channel.
 ====
-.. If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
 cluster.
-.. The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** The *DESIRED VERSION* indicates the cluster version that your cluster is running
 or is updating to.
 
-. Click *Updates Available*, select a version to update to, and click *Update*.
+. Click *Updates Available*, select
+ifdef::within[]
+a version to update to,
+endif::within[]
+ifdef::between[]
+the highest available version
+endif::between[]
+and click *Update*.
+The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
+the Operator upgrades on the *Cluster Operators* tab.
+
+ifdef::between[]
+. After the update completes, change the *CHANNEL* to the `stable-*` channel for the next minor version.
+
+. Click *Updates Available*, select the version that you want to update to, and click *Update*.
 The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
 the Operator upgrades on the *Cluster Operators* tab.
```

Comment:
This sentence is not between-specific.  Maybe it's best to leave the between-minor docs out of this module so it can stay focused on executing a single upgrade, and then have a different module or leading section on between-minors that says "to upgrade between minors, upgrade to the most recent release in your current channel by repeating {the upgrade proceedure} until you no longer have any available updates.  Then change your channel to the new `stable-*` and perform the available update."  Or something...

Reply:
How's this?